patches/woocommerce-subscriptions.0002.fix.cart-resubscribe-translation-type.patch

```diff
@@ -0,0 +1,26 @@
+From 9e3181c4d0ed230d09ad0f2108f7f753683a2042 Mon Sep 17 00:00:00 2001
+From: Bogdan Arizancu <bogdan@netzstrategen.com>
+Date: Wed, 30 Jun 2021 18:23:44 +0300
+Subject: [PATCH] Fixed woocommerce-subscriptions bug.
+
+---
+ includes/class-wcs-cart-resubscribe.php | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/includes/class-wcs-cart-resubscribe.php b/includes/class-wcs-cart-resubscribe.php
+index be724d7ae..9aae0639b 100644
+--- a/includes/class-wcs-cart-resubscribe.php
++++ b/includes/class-wcs-cart-resubscribe.php
+@@ -335,7 +335,8 @@ class WCS_Cart_Resubscribe extends WCS_Cart_Renewal {
+ 	public function order_button_text( $place_order_text ) {
+ 
+ 		if ( isset( WC()->cart ) && count( wcs_get_order_type_cart_items( 'resubscribe' ) ) === count( WC()->cart->get_cart() ) ) {
+-			$place_order_text = _x( 'Resubscribe', 'The place order button text while resubscribing to a subscription', 'woocommerce-subscriptions' );
++			// $place_order_text = _x( 'Resubscribe', 'The place order button text while resubscribing to a subscription', 'woocommerce-subscriptions' );
```

Comment:
Any chance to regenerate this without the commented out line?  Git has the history, so code doesn't need to be commented out.